packages/ui-alerts/src/Alert/__new-tests__/Alert.test.tsx

```diff
@@ -0,0 +1,271 @@
+    srdiv.setAttribute('aria-relevant', 'additions text')
+    srdiv.setAttribute('aria-atomic', 'false')
+    document.body.appendChild(srdiv)
+  })
+
+  afterEach(async () => {
+    srdiv?.parentNode?.removeChild(srdiv)
+    srdiv = null
+    consoleWarningMock.mockRestore()
+    consoleErrorMock.mockRestore()
+  })
+
+  it('should render', async () => {
+    render(<Alert variant="success">Success: Sample alert text.</Alert>)
+    const text = screen.getByText('Success: Sample alert text.')
+    expect(text).toBeInTheDocument()
+  })
+
+  describe('with generated examples', () => {
+    const generatedComponents = generateA11yTests(Alert, AlertExamples)
+
+    for (const component of generatedComponents) {
+      it(component.description, async () => {
+        const { container } = render(component.content)
+        const axeCheck = await runAxeCheck(container)
+        expect(axeCheck).toBe(true)
+      })
+    }
+  })
+
+  it('should not render the Close button when `renderCloseButtonLabel` is not provided', async () => {
+    render(<Alert variant="success">Success: Sample alert text.</Alert>)
+    const closeButton = screen.queryByRole('button')
+
+    expect(closeButton).not.toBeInTheDocument()
+  })
+
+  it('should call `onDismiss` when the close button is clicked with renderCloseButtonLabel', async () => {
+    const onDismiss = vi.fn()
+    render(
+      <Alert
+        variant="success"
+        renderCloseButtonLabel={<div>Close</div>}
+        onDismiss={onDismiss}
+      >
+        Success: Sample alert text.
+      </Alert>
+    )
+    const closeButton = screen.getByRole('button')
+
+    userEvent.click(closeButton)
+
+    await waitFor(() => {
+      expect(onDismiss).toHaveBeenCalled()
+    })
+  })
+
+  const iconComponentsVariants: Record<
+    NonNullable<AlertProps['variant']>,
+    string
+  > = {
+    error: 'IconNo',
+    info: 'IconInfoBorderless',
+    success: 'IconCheckMark',
+    warning: 'IconWarningBorderless'
+  }
+
+  ;(
+    Object.entries(iconComponentsVariants) as [
+      NonNullable<AlertProps['variant']>,
+      string
+    ][]
+  ).forEach(([variant, iconComponent]) => {
+    it(`"${variant}" variant should have icon "${iconComponent}".`, async () => {
+      const { container } = render(
+        <Alert variant={variant} transition="none">
+          Success: Sample alert text.
+        </Alert>
+      )
+      const icon = container.querySelector('svg[class$="-svgIcon"]')
+
+      expect(icon).toHaveAttribute('name', iconComponent)
+    })
+  })
+
+  it('should meet a11y standards', async () => {
+    const { container } = render(
+      <Alert variant="success" transition="none">
+        Success: Sample alert text.
+      </Alert>
+    )
+    const axeCheck = await runAxeCheck(container)
+    expect(axeCheck).toBe(true)
+  })
+
+  it('should add alert text to aria live region, when present', async () => {
+    const liver = document.getElementById('_alertLiveRegion')!
```

Comment:
Please rename this to liveRegion or something else instead of an anatomical terminology. :D

Reply:
I renamed liver to liveRegion everywhere